docs/conf.py

```diff
@@ -155,5 +155,48 @@ def setup(app):
 # This is the prefix common to all schema IDs in this repository
 asdf_schema_standard_prefix = "schemas"
 asdf_schema_reference_mappings = [
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/calibration_software_name-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/calibration_software_name-1.0.0.html",
+    ),
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/calibration_software_version-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/calibration_software_version-1.0.0.html",
+    ),
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/filename-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/filename-1.0.0.html",
+    ),
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/file_date-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/file_date-1.0.0.html",
+    ),
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/model_type-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/model_type-1.0.0.html",
+    ),
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/origin-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/origin-1.0.0.html",
+    ),
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/prd_version-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/prd_version-1.0.0.html",
+    ),
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/product_type-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/product_type-1.0.0.html",
+    ),
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/sdf_software_version-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/sdf_software_version-1.0.0.html",
+    ),
+    (
+        "asdf://stsci.edu/datamodels/roman/tags/telescope-1.0.0",
+        "https://rad.readthedocs.io/en/latest/generated/schemas/tagged_scalars/telescope-1.0.0.html",
+    ),
     ("asdf://stsci.edu/datamodels/roman/tags/", "https://rad.readthedocs.io/en/latest/generated/schemas/"),
+    ("asdf://stsci.edu/datamodels/roman/schemas/", "https://rad.readthedocs.io/en/latest/generated/schemas/"),
+    ("http://stsci.edu/schemas/asdf/", "https://asdf-standard.readthedocs.io/en/latest/generated/stsci.edu/asdf/"),
+    ("tag:stsci.edu:gwcs/wcs-*", "https://asdf-standard.readthedocs.io/en/latest/generated/stsci.edu/asdf/wcs/wcs-1.0.0.html"),
```

Comment:
```suggestion
    ("tag:stsci.edu:gwcs/wcs-*", "https://asdf-wcs-schemas.readthedocs.io/en/latest/generated/gwcs/wcs-1.0.0.html"),
```
I think this is the relevant wcs schema (for ASDF standard 1.5.0, ASDF standard 1.6.0 uses wcs-1.1.0).

I wonder if there is a way to create/manage these links that won't require manually updating them when schema/tag versions change. The tag wildcard is one issue here (see https://github.com/asdf-format/sphinx-asdf/issues/92).

For the other links I wonder if it's worth reorganizing the rad tags to match the schema layout (or schemas to match the tags). I think the reason the existing reference mapping isn't picking up the schemas is that the `tagged_scalars` schemas are prefixed with `.../schemas/tagged_scalars/` whereas the tags are `.../tags/...` (no tagged_scalars).